### PR TITLE
Replace Supabase by Appwrite for backend

### PR DESCRIPTION
**Pull request by Deepu178**
## Related Issue

- Information about the related issue

Closes: # 

### Describe the changes you've made
Replace Supabase by Appwrite for backend

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [x] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Code of Conduct

- [x ] I agree to follow this project's [Code of Conduct](https://github.com/UniKonf/vibey/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: src/lib/db/useAppwriteClient.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { Client } from 'appwrite';
+const client = new Client();
+const porjectId = process.env.APPWRITE_PROJECT_ID;
+
+client.setEndpoint('http://localhost/v1').setProject(`${porjectId}`);
```
Comment (Chandraprakash-Darji) [inline src/lib/db/useAppwriteClient.tsx:5]: here should be the deployed appwrite link?

Comment (Deepu178) [reply src/lib/db/useAppwriteClient.tsx:5]: yes but deployed only locally for now.

Comment (mkubdev) [reply src/lib/db/useAppwriteClient.tsx:5]: You can use [this kind of strategy](https://metaphore.vercel.app/stories/javascript/easy-fetching-on-nextjs) to build the string!

**Review by mkubdev — APPROVED** [review]
LGTM 😁

**Review by money8203 — APPROVED** [review]

File: src/lib/db/useAppwriteClient.tsx (hunk 1 above)
Comment (Deepu178) [reply src/lib/db/useAppwriteClient.tsx:5]: wow I found this very useful, I'll  implement from now.

**Pull request merged**